assets/lang/sklmessages_pl_PL.properties

```diff
@@ -39,7 +39,7 @@ game.status.idle=Bezczynny
 game.output.open=Otwórz okno konsoli
 game.output.lines=widocznych linii
 
-game.conflict.title=Ostrzeżenie o zduplikowanej instancji
+game.conflict.title=Ostrzeżenie o zduplikowanej instalacji
```

Comment:
Tutaj zauważyłem **błąd** przy drugiej edycji i go poprawiłem.

Reply:
Chociaż sam nie do końca wiem czy to błąd.

Reply:
Wiesz co, to nie jest błąd :D "Instance" po polsku brzmi "instancja". Chodzi tutaj o instancję, czyli w tym przypadku o kolejne wystąpienie tego samego programu (czy coś w tym stylu, nie znam dokładnej definicji). Jeśli np. na początku uruchomisz kalkulator, a później 2., 3. i 4., to one będą kolejnymi instancjami aplikacji Kalkulator.

Reply:
Wiesz co, to nie jest błąd :D "Instance" po polsku brzmi "instancja". Chodzi tutaj o instancję, czyli w tym przypadku o kolejne wystąpienie tego samego programu (czy coś w tym stylu, nie znam dokładnej definicji). Jeśli np. na początku uruchomisz kalkulator, a później 2., 3. i 4., to one będą kolejnymi instancjami aplikacji Kalkulator.

Reply:
Wiesz co, to nie jest błąd :D "Instance" po polsku brzmi "instancja". Chodzi tutaj o instancję, czyli w tym przypadku o kolejne wystąpienie tego samego programu (czy coś w tym stylu, nie znam dokładnej definicji). Jeśli np. na początku uruchomisz kalkulator, a później 2., 3. i 4., to one będą kolejnymi instancjami aplikacji Kalkulator.

Reply:
Okej, rozumiem już o co chodzi. Już poprawiam.

Reply:
Poprawiłem @Greg-21